tests/h/nipsa/search_test.py

```diff
@@ -1,11 +1,51 @@
 # -*- coding: utf-8 -*-
 
+import mock
+import pytest
+
 from h.nipsa import search
 
 
-def test_nipsa_filter_filters_out_nipsad_annotations():
+@pytest.mark.usefixtures('group_service')
+class TestFilter(object):
+    def test_call_returns_nipsa_filter(self, pyramid_request, nipsa_filter):
+        f = search.Filter(pyramid_request)
+
+        assert f({}) == nipsa_filter.return_value
+
+    def test_call_passes_group_service(self, pyramid_request, nipsa_filter, group_service):
+        f = search.Filter(pyramid_request)
+
+        f({})
+
+        nipsa_filter.assert_called_once_with(group_service, mock.ANY)
+
+    def call_passes_request_user(self, pyramid_request, nipsa_filter):
```

Comment:
This code isn't being executed because the test method doesn't start with `test_`.